### PR TITLE
Implement organizer context

### PR DESCRIPTION
**Pull request by qdozaq**
Creating the organizer context and hook.

Think I got all the places that need org data. Let me know if I missed anything.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -44,49 +46,70 @@ function PrivateRoute({ children, ...rest }) {
...
+// grab from domain or some service
+const ORGANIZATION_ID = "1";
+
 function App() {
+  const [org, setOrg] = useState();
+  useEffect(() => {
+    Organization.init(ORGANIZATION_ID)
+      .then((org) => setOrg(org))
+      .catch((error) => {
+        console.error(error);
```
Comment (mat10tng) [inline src/App.jsx:58]: should we do a snackbar error here ?

File: src/App.jsx [hunk 2]
```diff
@@ -44,49 +46,70 @@ function PrivateRoute({ children, ...rest }) {
...
-              </Switch>
-              <Snackbar.Context.SnackbarConsumer>
-                {(value) => {
-                  return <Snackbar handleClose={value.closeSnackbar} {...value.snackbar} />;
-                }}
-              </Snackbar.Context.SnackbarConsumer>
-            </Snackbar.Context.SnackbarProvider>
+            <OrganizationContext.Provider value={org}>
+              <Snackbar.Context.SnackbarProvider>
+                {org === null && <Redirect to="/404" />}
```
Comment (mat10tng) [inline src/App.jsx:75]: I think there is some new way of handle routing, by Amy

File: src/App.jsx (hunk 1 above)
Comment (qdozaq) [reply src/App.jsx:65]: Hmm there's not currently a good way to set the snackbar while outside of the snackbar provider. Something to look at later

Comment (jwu910) [reply src/App.jsx:65]: Depends on how the app should respond to the app unable to run the init org method right?

If we’re dependent on these org hook being set in the provider, maybe we need a more serious error lol. 

Will the app work if the org is not set?

Comment (qdozaq) [reply src/App.jsx:65]: Everything basically depends on the org being set in order to make certain calls. So right now I just have it redirecting to the organizer signup page if it doesn't exist

**Pull request merged**